opr/functions.py

```diff
@@ -137,21 +137,22 @@ def e260_ssnn_calc(sequence):
         e260 -= BASE_EXTINCTION_COEFFICIENTS[base]
     return e260
 
-
-def protein_seq_calc(rna_sequence, frame):
+def protein_seq_calc(rna_sequence, frame=1, one_letter=True):
     """
     Calculate the sequence of amino acids from a mRNA sequence.
 
     :param rna_sequence: mRNA sequence
     :type rna_sequence: str
     :param frame: frame of translation (1, 2, or 3)
     :type frame: int
+    :param one_letter: Whether to return amino acids in 1-letter codes (True) or 3-letter codes (False).
```

Comment:
- make the first character lower case and remove `.` for consistency.

Also, I suggest changing the parameter name to `multi_letter` (and invert the logic).